### PR TITLE
[ALLUXIO-1782] Add logging to the set attribute command in BaseFileSystem

### PR DESCRIPTION
**Pull request by saltylin**
https://alluxio.atlassian.net/browse/ALLUXIO-1782

### CODE REVIEW

File: core/client/src/main/java/alluxio/client/file/BaseFileSystem.java [hunk 1]
```diff
@@ -284,6 +284,7 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     try {
       masterClient.setAttribute(path, options);
+      LOG.info("Set attributes of file {} to ({})", path.getPath(), generateLogInfo(options));
```
Comment (gpang) [inline core/client/src/main/java/alluxio/client/file/BaseFileSystem.java]: I think SetAttributeOptions already has a `toString()`, so I don't think you need to add a `generateLogInfo()` method.

Also, I think you can use "Set attributes for path {} with options {}"

**Comment by gpang** [comment]
Thanks for the PR @saltylin ! I left a minor comment in the code.

File: core/client/src/main/java/alluxio/client/file/BaseFileSystem.java (hunk 1 above)
Comment (saltylin) [inline core/client/src/main/java/alluxio/client/file/BaseFileSystem.java]: SetAttributeOptions#toString method simply transfers unset attributes to "null" string, and I just thought it's not readable.

**Comment by saltylin** [comment]
@gpang Thanks for you review. I just thought the string SetAttributeOptions#toString method returns is not very readable, and I'm not sure is it suitable to be logged.

**Comment by saltylin** [comment]
@gpang Updated! Use toString() method instead.

**Comment by saltylin** [comment]
Jenkins, test this please.

**Comment by gpang** [comment]
Thanks for the improvement!

**Comment by gpang** [comment]
LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**